en/docs/references/org-domains-urls.md

```diff
@@ -1,24 +1,80 @@
-# Organization domains and URLs
+# Organization-Specific REST API Routing
 
 ## Organization URLs
 
-As a part of the B2B Organization Management feature, incorporated the organization identifier as first-class support for the URLs (similar to the current tenant specific endpoints) by introducing a path param to denote the specific organization to which the request should be directed to.
+As a part of the B2B Organization Management feature, incorporated the organization identifier as first-class support for the URLs by introducing a path param to denote the specific organization to which the request should be directed to.
 
-URL format - o/{organization id}/…
+URL format - `o/{organization id}/…`
 
-Organization-specific routing capability is introduced. The BasicAuthenticationHandler has also been improved to handle authentication when the username has the organization domain(id) appended.
+Organization-specific routing capability is introduced. Here we are supporting both Basic Authentication and OAuth 2.0 flows.
```

Comment:
Organization-specific routing in WSO2 CIAM Cloud supports both basic authentication and OAuth 2.0 flows as explained below.